pkgs/development/libraries/gdal/default.nix

```diff
@@ -20,14 +20,13 @@ composableDerivation.composableDerivation {} (fixed: rec {
 
   hardeningDisable = [ "format" ];
 
-  # Don't use optimization for gcc >= 4.3. That's said to be causing segfaults.
   # Unset CC and CXX as they confuse libtool.
-  preConfigure = "export CFLAGS=-O0 CXXFLAGS=-O0; unset CC CXX";
```

Comment:
The commit that introduced the changes has an explanation https://github.com/NixOS/nixpkgs/commit/b3d33d398c23295d10cf79448e549a034b6d619e

/cc @benley